InteractiveHtmlBom/web/render.js

```diff
@@ -498,10 +515,12 @@ function prepareCanvas(canvas, flip, transform) {
   ctx.scale(transform.zoom, transform.zoom);
   ctx.translate(transform.panx, transform.pany);
   if (flip) {
-    ctx.scale(-1, 1);
+    ctx.rotate(deg2rad(settings.boardRotation + 180.));
+  } else {
+    ctx.rotate(deg2rad(settings.boardRotation));
```

Comment:
Is the board being upside down intentional?

To avoid breaking rendering for "normal" ibom pages this should be parametrised in pcbdata.

Reply:
I reverted the changes here; just realized I could have commanded `pcb-rnd` to vertically flip the PCB to render the back side. Previously, it was horizontal flipped.